modules/server/src/main/ts/bedrock/auto/RemoteDriver.ts

```diff
@@ -4,6 +4,8 @@ import * as deepmerge from 'deepmerge';
 import { DeviceFarmClient, CreateTestGridUrlCommand } from '@aws-sdk/client-device-farm';
 import { Driver, DriverSettings } from './Driver';
 
+export const REMOTE_IDLE_TIMEOUT_SECONDS = 360;
```

Comment:
I wonder if this is something that we might want to configure?

Reply:
I think that would make configuring bedrock too complex.

LambdaTest defaults to 100 seconds, and this is hard coded (then referenced elsewhere) to make sure we don't time out the LambdaTest run while waiting for our configurable timeouts.